### PR TITLE
[CI:DOCS] Fix up example description of podman-inspect.1.md.in

### PR DESCRIPTION
**Pull request by rhatdan**
#### Does this PR introduce a user-facing change?

```release-note
None
```

### CODE REVIEW

File: docs/source/markdown/podman-inspect.1.md.in [hunk 1]
```diff
@@ -122,43 +122,43 @@ Inspect the fedora image
 ]
 ```
 
-Inspect the specified image for Image Name.
+Inspect the specified image for Image Name:
```
Comment (vrothberg) [inline docs/source/markdown/podman-inspect.1.md.in:125]:
```suggestion
Inspect the specified image with the `ImageName` format specifier:
```

Comment (vrothberg) [reply docs/source/markdown/podman-inspect.1.md.in:125]: ? Maybe the other cases below as well

**Review by vrothberg — COMMENTED** [review]
Would it makes sense to group the examples using `--format`?

File: docs/source/markdown/podman-inspect.1.md.in [hunk 2]
```diff
@@ -122,43 +122,43 @@ Inspect the fedora image
...
 ```
 # podman inspect a04 --format "{{.ImageName}}"
 fedora
 ```
 
-Inspect the specified image for GraphDriver Name.
+Inspect the specified image for GraphDriver Name:
```
Comment (TomSweeneyRedHat) [inline docs/source/markdown/podman-inspect.1.md.in:131]:
```suggestion
Inspect the specified image for GraphDriver Name field:
```

**Comment by rhatdan** [comment]
@vrothberg @TomSweeneyRedHat @edsantiago PTAL
@containers/podman-maintainers PTAL

**Pull request merged**